deploy/infrastructure/cli/deploy.ps1

```diff
@@ -4,7 +4,10 @@ param (
     [string]$resourceGroupCount = 1,
     [Parameter(Mandatory = $true)]
     [string]$subscriptionId,
-    [string]$deploymentType = "aca"
+    [string]$deploymentType = "aca",
+    [string]$skipAzureCosmosVectorDB = "false",
+    [string]$adminUsername = "admin",
+    [securestring]$adminPassword = "admin1234"
```

Comment:
This will definitely flag GH secret scanner. I'm surprised it already did not.

Reply:
Updated it.